packages/shared/src/test/setup.ts

```diff
@@ -20,10 +20,14 @@ vi.mock('@react-native-firebase/crashlytics', () => {
 });
 
 export function mockUrbit() {
-  vi.mock('../api/urbit', async () => {
-    return {
+  vi.mock('../api/urbit', async (importOriginal) => {
+    const mod = await importOriginal<typeof import('../api/urbit')>();
```

Comment:
this form preserves the type of the original import, so mocked implementations here are sure to match the original ones.

it's notable that this changes behavior of tests a bit – since we splat `...mod`, we now include all of the original members of `../api/urbit`. This is different from the previous code, which would error if a test called a member of that module that was not mocked. I see the benefit of that approach, but I wasn't sure if it was intentional. I think we can restore that functionality by just deleting line 26. wdyt?